src/redux/configureStore.js

```diff
@@ -1,10 +1,12 @@
 import { configureStore } from '@reduxjs/toolkit';
 import logger from 'redux-logger';
 import reducersRockets from './rockets/rockets';
+import missionReducer from './missions/missions';
 
 const store = configureStore({
   reducer: {
     rockets: reducersRockets,
+    missionReducer,
```

Comment:
For your missions Reducer, I think it's best you create a separate key for it since the rocket reducer is on a different file.it's going to be best practice to make it neat and understandable.
Something like this:
`const store = configureStore({
  reducer: {
    rockets: reducersRockets,
    missions: missionReducer,
    `
That will do.